### ISSUE TITLE
Channel Settings: banner image and avatar image

### ISSUE THREAD

**Issue by pdpinch** [mitodl/open-discussions#927]
As a channel owner, I'd like to brand my channel using a custom banner for the page header on my channel and a custom avatar. The avatar appears in the header and is also used in the navigation menu. 

See https://invis.io/SVMZO4WWCT9#/301262024_Channel_ClimateX for wireframes of navigation menu, channel header. You can get final designs from Lucas later this week. 

See https://projects.invisionapp.com/d/main#/console/14224393/303979448/preview for wireframes of channel settings. You can get final designs from Lucas later in the sprint. 

### acceptance criteria:
- [x] add channel banner and channel avatar to channel model
- [x] add forms to the channel appearance setting to upload the banner image and the avatar image

### related issues
- #1061 update the navigation to use the channel avatars
- #989 update the channel listing template to use the banner and avatar
- channels with no banner will have a white background
- channels with no avatar will have an avatar based off their initials

### questions
- [ ] what color(s) should be used for channels with no custom avatar?

**Comment by pdpinch** [comment]
Moving this to In Progress

**Comment by noisecapella** [comment]
Do we know what the dimensions are for the channel avatar and banner?

**Comment by pdpinch** [comment]
I broke out the channel template into another issue (#989) and updated the InVision link in the description.

**Comment by pdpinch** [comment]
I updated the description to move the changes to the left nav into a separate issue. 

We can close this after #983 merges.
